styleguide_example/files/tests/flows/test_standard_upload.py

```diff
@@ -5,8 +5,10 @@
 
 from rest_framework.test import APIClient
 
+from config.settings.files_and_storages import FILE_MAX_SIZE
```

Comment:
1. We don't need to use settings directly, check how to override settings in Django tests.
2. In the case where you need to use a setting directly, never import it from a module. Use `django.conf.settings` instead.